cmsplugin_articles_ai/models/articles.py

```diff
@@ -153,6 +153,9 @@ class Article(PublisherModel):
     modified_at = models.DateTimeField(
         _("last modified"), auto_now=True, editable=False,
     )
+    show_lead = models.BooleanField(
+        _("show lead"), default=True, help_text=_("Show lead text."),
```

Comment:
Is that a plugin template or what? I'm just a bit hesitant to add such field to the model to avoid bloating the model with pretty random fields.. Also it's representational logic which should be handled by the render logic instead of the model IMO.

Feel free to add lead_text property or even a field, but the proposed `show_lead` doesn't sound good to me.